background/ncn_firmware.md

```diff
@@ -0,0 +1,34 @@
+# Firmware
+
+This page describes the non-compute node's (NCN) firmware; the minimum versions, and how to interegate information
+from an NCN.
```

Comment:
The machine (s) are being interrogated, information is being extracted and reviewed by an administrator or other authority to the system.

Also used the word because in some contexts the BMCs affiliation is unknown; whether it's an NCN, CN, UAN, or other.

Reply:
It'll make more sense when the actual steps for interrogation are here (MTL-1972).